src/main.rs

```diff
@@ -0,0 +1,38 @@
+mod constants;
+mod conversation;
+mod db;
+mod embeddings;
+mod github;
+mod prelude;
+mod routes;
+mod utils;
+use std::{path::Path, sync::Arc};
+
+use actix_web::{web, App, HttpResponse, HttpServer};
+use constants::ACTIX_WEB_SERVER_PORT;
+use env_logger::Env;
+use tracing_actix_web::TracingLogger;
+
+#[actix_web::main]
+async fn main() -> std::io::Result<()> {
+    dotenv::dotenv().ok();
+
+    env_logger::init_from_env(Env::default().default_filter_or("info"));
```

Comment:
Can we add this logging env variable to the `.env.example` file so contributors can easily see what options are possible for logging?

Reply:
https://github.com/open-sauced/repo-query/pull/1/commits/3ec893a487f6c806e750ec5e8e398d81024d9fd6